mono/metadata/w32subset.h

```diff
@@ -178,3 +178,43 @@
 #define HAVE_API_SUPPORT_WIN32_GET_MODULE_HANDLE 0
 #endif
 #endif
+
+#ifndef HAVE_API_SUPPORT_WIN32_GET_THREAD_CONTEXT
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT) || G_HAVE_API_SUPPORT(HAVE_UWP_WINAPI_SUPPORT)
+#define HAVE_API_SUPPORT_WIN32_GET_THREAD_CONTEXT 1
+#else
+#define HAVE_API_SUPPORT_WIN32_GET_THREAD_CONTEXT 0
+#endif
+#endif
+
+#ifndef HAVE_API_SUPPORT_WIN32_SET_THREAD_CONTEXT
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+#define HAVE_API_SUPPORT_WIN32_SET_THREAD_CONTEXT 1
+#else
+#define HAVE_API_SUPPORT_WIN32_SET_THREAD_CONTEXT 0
+#endif
+#endif
+
+#ifndef HAVE_API_SUPPORT_WIN32_CANCEL_SYNCHRONOUS_IO
+#if G_HAVE_API_SUPPORT(HAVE_CLASSIC_WINAPI_SUPPORT)
+#define HAVE_API_SUPPORT_WIN32_CANCEL_SYNCHRONOUS_IO 1
+#else
+#define HAVE_API_SUPPORT_WIN32_CANCEL_SYNCHRONOUS_IO 0
+#endif
+#endif
+
+#ifndef HAVE_API_SUPPORT_WIN32_OPEN_THREAD
```

Comment:
OpenThread is supported now. In order to work with 10240, guidance is declare it yourself. So then this isn't needed.

Reply:
Will keep since this we have users supporting older UWP versions and would like to use this file.